runtime/src/registry.rs

```diff
@@ -348,7 +379,7 @@ mod test {
 
     #[test]
     /// Test that store::Orgs::decode_key after store::Orgs::storage_map_final_key
-    /// is identify as to the original input id.
+    /// is identified as the original input id.
```

Comment:
this was my typo and its meant to be "identity":

```suggestion
    /// is identity as to the original input id.
```

Reply:
I don't think this is a valid english sentence. It should be `identical` or `equal`.

Reply:
Yeah, you might be right. What it tries to say is that `input == store::Orgs::decode_key · store::Orgs::storage_map_final_key $ input`.